logging/elektronFormatter.go

```diff
@@ -0,0 +1,36 @@
+package logging
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type elektronFormatter struct {
+	TimestampFormat string
+}
+
+func (f elektronFormatter) Format(entry *log.Entry) ([]byte, error) {
```

Comment:
Does the formatter adhere to the format specified in the config provided?

Reply:
No, actually as per our discussion earlier you had suggested to implement a decorator pattern for using the format specified in config. So currently, it  has a fixed format and there is technically no use of the format specified in config file.